modules/doc/content/ncrc/applications/ncrc_develop.md.template

```diff
@@ -0,0 +1,147 @@
+of modules.
+
+- If +Personal Machine+: Follow the [Conda Installation](installation/install_miniconda.md) instructions and then perform the following:
+
+  ```bash
+  mamba create -n moose moose-tools moose-libmesh
+  ```
+
+- If +[!ac](INL) [!ac](HPC) Sawtooth or Lemhi+ (required each time you log in):
+
+  ```bash
+  module load use.moose moose-dev
+  ```
+
+For those operating on other [!ac](HPC) clusters, you should contact that system's administrator,
+and use their recommended compiler stack. It will need to satisfy to our
+[minimum requirements](sqa/minimum_requirements.md).
+
+!alert warning title=Conda packages may be suboptimal on HPC
+Conda packages have the advantage of being able to work on most machines, but on [!ac](HPC)
+clusters, this flexibility potentially comes at the cost of the optimizations developed by
+[!ac](HPC) administrators.
+
+## Cloning {{ApplicationName}} id=cloning
+
+Instructions for obtaining a proper clone of {{ApplicationName}} can be found at
+[NCRC/{{ApplicationName}}](ncrc/applications/ncrc_level4_{{ApplicationLower}}.md).
+
+## Building {{ApplicationName}} id=build
+
+Building {{ApplicationName}} begins with applying upstream changes, updating your submodules and/or
+Conda packages, and finally running `make`. Before we begin, +remember these two important
+warnings+:
+
+!alert warning title=You updated Conda?
+`Then you should also perform a submodule update`.
+
+!alert warning title=You performed a submodule update?
+`Then you should also perform a Conda update or rebuild PETSc/libMesh`.
+
+The symmetry importance comes from our Conda packages and support libraries being version controlled
+by and within the MOOSE repository. Updates to MOOSE (the `{{binary}}/moose` submodule) requires you
+to stay on top of any Conda updates (if Personal Machine), or rebuilding PETSc and libMesh (if using
+[!ac](HPC) resources).
+
+### Applying Upstream Changes
+
+If this is your first time following these instructions there should be no updates. However, we will
+perform an update to familiarize yourself with the entire process.
+
+{{ApplicationName}} developers will expect users to have performed the following sections, whenever
+they says "I have updated {{ApplicationName}}".
+
+```bash
+cd ~/projects/{{binary}}
+git fetch upstream
+```
+
+`git fetch upstream` updates your local references with that of the remote. It does not actually
+modify any source files. It will return nothing if there is nothing new to update.
+
+You need to know your upstream head branch. This will be the branch you will want to continuously
+rebase against. That is, keeping your local {{binary}} repository up to date. The following command
+will reveal the upstream head branch name:
+
+```bash
+git remote show upstream | grep "HEAD branch" | sed 's/.*: //'
+```
+
+Next, we will perform a `rebase` against the provided upstream branch name:
+
+```bash
+git rebase upstream/branch   # replace 'branch' with the discovered branch name
```

Comment:
I think put your comment in the text instead. I bet some people will copy and paste it...

Reply:
Yeah I can make that text instead.